packages/react-native/ReactCommon/react/runtime/ReactInstance.cpp

```diff
@@ -458,7 +459,8 @@ void ReactInstance::initializeRuntime(
 
     timerManager_->attachGlobals(runtime);
 
-    bindingsInstallFunc(runtime);
+    auto callInvoker = std::make_shared<RuntimeSchedulerCallInvoker>(runtimeScheduler_);
```

Comment:
We should able to get to the existing one and avoid allocating a new one.

Reply:
How? Do you have any pointers for me?

Reply:
I'll have a look. I assumed this would be accessible from Reactinstance, since C++ TurboModules get it, but I may be wrong.